frontend/src/features/missions/MissionSchema.ts

```diff
@@ -101,37 +101,76 @@ const InfractionSchema: Yup.SchemaOf<Infraction> = Yup.object().shape({
   vesselType: Yup.mixed().oneOfOptional(Object.values(VesselTypeEnum))
 })
 
-const EnvActionControlSchema: Yup.SchemaOf<EnvActionControl> = Yup.object()
-  .shape({
-    actionNumberOfControls: Yup.number().required('Requis'),
-    actionStartDateTimeUtc: Yup.string().nullable().required('Requis'),
-    actionTargetType: Yup.string().nullable().required('Requis'),
-    actionType: Yup.mixed().oneOf([ActionTypeEnum.CONTROL]),
-    geom: Yup.array().ensure(),
-    id: Yup.string().required(),
-    infractions: Yup.array().of(InfractionSchema).ensure().required(),
-    themes: Yup.array().of(ThemeSchema).ensure().required(),
-    vehicleType: Yup.string().when('actionTargetType', (actionTargetType, schema) => {
-      if (!actionTargetType || actionTargetType === ActionTargetTypeEnum.VEHICLE) {
-        return schema.nullable().required('Requis')
-      }
-
-      return schema.nullable()
+const getEnvActionControlSchema = (ctx: any): Yup.SchemaOf<EnvActionControl> =>
+  Yup.object()
+    .shape({
+      actionNumberOfControls: Yup.number().required('Requis'),
+      actionStartDateTimeUtc: Yup.string()
+        .nullable()
+        .required('Requis')
+        .test('isDateBefore', 'La date doit être postérieure à la date de début de mission', value =>
+          value ? !(new Date(value) < new Date(ctx.from[1].value.startDateTimeUtc)) : true
```

Comment:
Comme tu préfères mais sache que tu peux utiliser `isAfter()` / `isBefore()` : https://github.com/MTES-MCT/monitorfish/blob/ivan/add-date-inputs-validation-in-mission-form/frontend/src/features/SideWindow/MissionForm/MainForm/schemas.ts#L16